### PR TITLE
Updated Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
:-)

### CODE REVIEW

File: po/nl.po [hunk 1]
```diff
@@ -184,7 +183,7 @@ msgid "Open an image"
 msgstr "Afbeelding openen"
 
 #: src/ui/app-menus.ui:88 src/ui/win-menus.ui:145
-#, fuzzy
```
Comment (maoschanz) [inline po/nl.po]: really?

File: po/nl.po [hunk 2]
```diff
@@ -718,15 +717,15 @@ msgid "Default background"
...
 #: src/preferences.py:79
-msgid ""
-"JPEG and BMP images can't handle transparency. If you save your images in "
-"these formats, what do want to use to replace transparent pixels?"
-msgstr ""
+msgid "JPEG and BMP images can't handle transparency. If you save your images in 
+these formats, what do want to use to replace transparent pixels?"
+msgstr "JPEG- en BMP-afbeeldingen kunnen niet omgaan met doorzichtigheid. Als je 
+afbeeldingen opslaan in deze formaten, wat wil je dan gebruiken om "
+doorzichtige pixels te vervangen?"
```
Comment (maoschanz) [inline po/nl.po]: missing quote

File: po/nl.po [hunk 3]
```diff
@@ -737,13 +736,16 @@ msgid "Black"
 msgstr "Zwart"
 
 #: src/preferences.py:84 src/utilities.py:138
-#, fuzzy
+
 msgid "Checkboard"
-msgstr "Toetsenbord"
+msgstr "Schaakbord"
```
Comment (maoschanz) [inline po/nl.po]: not sure what happened here (1/2)

File: po/nl.po [hunk 4]
```diff
@@ -737,13 +736,16 @@ msgid "Black"
...
 
 #: src/preferences.py:84
 msgid "Ask before saving"
-msgstr ""
+msgstr "Vragen alvorens op te slaan"
+
+msgid "Checkboard"
+msgstr "Toetsenbord"
```
Comment (maoschanz) [inline po/nl.po]: not sure what happened here (2/2)

**Comment by Vistaus** [comment]
Woah, wtf happened? This is the first time I encounter issues like this with po files lol

I will take a look at cleaning up this mess :)

**Comment by Vistaus** [comment]
Created a new pull request which looks to be better: #128

**Pull request closed without merging**